tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/GitStore.cs

```diff
@@ -138,15 +141,34 @@ public class GitStore : IAssetsStore
 
             if (pendingChanges.Length > 0)
             {
-                // TODO: Azure/azure-sdk-tools/3698
+                consoleWrapper.WriteLine("There are pending git chances, are you sure you want to reset? [Y|N]");
+                while (true)
+                {
+                    string response = consoleWrapper.ReadLine();
+                    response = response.ToLower();
```

Comment:
```suggestion
                    response = response.ToLowerInvariant();
```

Reply:
Nit. Where possible it's preferred to use invariant.

Reply:
It will be updated with the next commit.